CHANGELOG_PENDING.md

```diff
@@ -45,15 +45,14 @@ program](https://hackerone.com/tendermint).
 
 - Go API
 
-  - [libs/pubsub][\#4070](https://github.com/tendermint/tendermint/pull/4070) `Query#(Matches|Conditions)` returns an error.
   - [rpc/client] \#3471 `Validators` now requires two more args: `page` and `perPage`
```

Comment:
Just for my own understanding.... what is the status of things in CHANGELOG_PENDING? What has to happen for an entry to persist in this file even after a release?

Reply:
this hasn't happened in the past, but since we merged major-releases PRs then these things were not included in the release and should stay awaiting the day they are released. 😄

Reply:
got it! thanks.